openproblems/tasks/cell_cell_communication/methods/logfc.py

```diff
@@ -0,0 +1,21 @@
+from ....tools.decorators import method
+from .liana import liana
```

Comment:
This method can probably just be moved to `liana.py`, and then they can also share a decorator partial.

Reply:
I'm not sure I follow. Do you mean to move all of the code to `liana.py` so that I don't re-import the dependencies for each method?

Reply:
Since all of these methods are using liana, they all share key aspects of the decorator. This code appears basically identically in each of the files:
```
_***_method = functools.partial(
    method,
    paper_name="Comparison of methods and resources for cell-cell "
    "communication inference from single-cell RNA-Seq data",
    paper_url="https://www.nature.com/articles/s41467-022-30755-0",
    paper_year=2022,
    code_url="https://github.com/saezlab/liana",
    image="openproblems-r-extras",
)
```

Since they all use the same base code, I'd just put them all in liana.py a) because it's more clear and b) so you can re-use the decorator

Reply:
Yep. Agreed :) Will do that!

Reply:
Moved the methods to liana.py, but I'm not sure if we should completely get rid of the decorators. The only parts that all of them share is the `code_url` and `image` params, while `paper_*` parameters are unique.

Reply:
That's the point of a partial :) They can share the common stuff and not share the unique stuff. I added a commit to this effect